docs/installation.md

```diff
@@ -138,32 +159,46 @@ gcp:
   google_project: Give your project a name 
 ```
 
+Add your custom plans to the `config.yml` file, for example, plans for MySQL
+
+```yaml
+service:
+  csb-google-mysql:
+    plans: '[{"name":"default","id":"eec62c9b-b25e-4e65-bad5-6b74d90274bf","description":"Default MySQL v8.0 10GB storage","metadata":{"displayName":"default"},"mysql_version":"MYSQL_8_0","storage_gb":10,"tier":"db-n1-standard-2"}]'
+```
+
 ### Push and Register the Broker
 
 Push the broker as a binary application:
 
 ```bash
-make push-broker
-```
-
-Once this completes, the output from `cf marketplace` should include:
+SECURITY_USER_NAME=someusername
+SECURITY_USER_PASSWORD=somepassword
+APP_NAME=cloud-service-broker
 
+chmod +x cloud-service-broker
+cf push "${APP_NAME}" -c './cloud-service-broker serve --config config.yml' -b binary_buildpack --random-route --no-start
 ```
-csb-google-mysql            small, medium, large   MySQL is a fully managed service for the Google Cloud Platform.
 
-csb-google-postgres         small, medium, large   PostgreSQL is a fully managed service for the Google Cloud Platform.
-
-csb-google-redis            basic, ha              Cloud Memorystore for Redis is a fully managed Redis service for the Google Cloud Platform. 
+Bind the MySQL database and start the service broker:
+```bash
+cf bind-service cloud-service-broker csb-sql
+cf start "${APP_NAME}"
+```
 
-csb-google-storage-bucket   default                Google Cloud Storage that uses the Terraform back-end and grants service accounts IAM permissions directly on the bucket.      
+Register the service broker:
+```bash
+BROKER_NAME=csb-$USER
 
-csb-google-bigquery         standard               A fast, economical and fully managed data warehouse for large-scale data analytics.   
+cf create-service-broker "${BROKER_NAME}" "${SECURITY_USER_NAME}" "${SECURITY_USER_PASSWORD}" https://$(cf app "${APP_NAME}" | grep 'routes:' | cut -d ':' -f 2 | xargs) --space-scoped || cf update-service-broker "${BROKER_NAME}" "${SECURITY_USER_NAME}" "${SECURITY_USER_PASSWORD}" https://$(cf app "${APP_NAME}" | grep 'routes:' | cut -d ':' -f 2 | xargs)
```

Comment:
We detected the routes get not correctly parsed if the default term language is not English, the command needs to be

cf create-service-broker "${BROKER_NAME}" "${SECURITY_USER_NAME}" "${SECURITY_USER_PASSWORD}" https://$(LANG=EN cf app "${APP_NAME}" | grep 'routes:' | cut -d ':' -f 2 | xargs) --space-scoped || cf update-service-broker "${BROKER_NAME}" "${SECURITY_USER_NAME}" "${SECURITY_USER_PASSWORD}" https://$(LANG=EN cf app "${APP_NAME}" | grep 'routes:' | cut -d ':' -f 2 | xargs)

Reply:
Thank you very much :heart: